src/lib.rs

```diff
@@ -76,6 +76,8 @@ pub use serde_json_core;
 
 pub use derive_miniconf::{Miniconf, MiniconfAtomic};
 
+pub use heapless;
```

Comment:
If we could keep `heapless` out of the public API, I think that may be beneficial. That helps ease dependency issues later